src/transient.rs

```diff
@@ -691,3 +691,20 @@ mod either_impls {
         type Transience = (Covariant<L>, Covariant<R>);
     }
 }
+
+#[cfg(feature = "serde_json")]
+mod serde_json_impls {
+    use crate::*;
```

Comment:
I'd prefer to either use a specific import of `crate::Static` here, or to not import it and just prefix the paths directly, instead of using a wildcard import. It doesn't _really_ make a difference, but feels cleaner IMO.